L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -121,13 +121,25 @@
 _phase2_siml1emulator.add(l1tNNCaloTauEmulator)
 
 # ---- Produce the emulated CaloJets and Taus
-from L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cfi import *
+from L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cff import *
```

Comment:
This change broke the emulation. As far as I can tell there is no `l1tPhase2CaloJetEmulator_cff`, but `l1tPhase2CaloJetEmulator_cfi` does exist.

Reply:
Why was this change, and the `L1TCaloJetsTausTask` added at all for this PR?

Reply:
I coppied the same file we had in l1t-offline, beleiving that are in sync. There the name is cff (https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-1400pre3_v8/L1Trigger/Configuration/python/SimL1Emulator_cff.py )

Reply:
Please don't develop on top of the prototype, there tends to be things in there that are not final or CMSSW official, just cherry-pick the commits directly for PRs.

I'll go hunt down whoever made this change in our repository and understand if it is "ahead of" or "behind" CMSSW to understand if there is another PR we can expect to make this change. Even if it is ultimately "correct" this will introduce merge conflicts though.

Reply:
ok thanks I will fix it.

Best

Reply:
This change was ultimately introduced in https://github.com/cms-sw/cmssw/pull/44571. @gkaratha I think the best choice of action is to just recreate the PR by cherry picking the commits and force pushing here.